sentry_sdk/scrubber.py

```diff
@@ -145,9 +145,17 @@ def scrub_breadcrumbs(self, event):
         with capture_internal_exceptions():
             if "breadcrumbs" in event:
                 if "values" in event["breadcrumbs"]:
-                    for value in event["breadcrumbs"]["values"]:
-                        if "data" in value:
-                            self.scrub_dict(value["data"])
+                    if (
+                        isinstance(event["breadcrumbs"]["values"], AnnotatedValue)
```

Comment:
Here you check if `event["breadcrumbs"]["values"]` is an `AnnotatedValue` but above in client.py you set `event["breadcrumbs"]` to be an `AnnotatedValue`. Can you help me understand?